tests/instances.spec.ts

```diff
@@ -247,3 +247,54 @@ test("Create an image from an instance", async ({ page }) => {
   const imageName = await getImageNameFromAlias(page, imageAlias);
   await deleteImage(page, imageName);
 });
+
+test("Bulk start, pause, unpause and stop instances", async ({ page }) => {
+  await page.goto("/ui/");
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search and filter").fill(instance);
+  await page.getByText("Search for").click();
```

Comment:
This doesn't work for Firefox and lets the test suite fail.
I searched around our existing tests and found this bit confirming the search for volumes, I assume it would work on both chrome and firefox:
```
  await page.getByPlaceholder("Search and filter").fill(volume);
  await page.getByPlaceholder("Search and filter").press("Enter");
  await page.getByPlaceholder("Add filter").press("Escape");
```

Reply:
Thank you very much, David. I am not sure if this addressed the problem that I found when running Playwright against firefox, locally however I later found out that for some reason, Firefox and Chromium do not have the same locators for the checkbox. However, this was circumvented when I selected the dropdown option instead and bulk selected the instances on the page, that way. Please let me know your thoughts on the new implementation.